packages/@okta/vuepress-site/code/android/index.md

```diff
@@ -36,7 +36,7 @@ This method works with both Okta Identity Engine and Okta Classic Engine.
 
 SDK: [Okta Mobile SDK for Kotlin](https://github.com/okta/okta-mobile-kotlin).
 
-Example: [Browser Sign In](https://github.com/okta/samples-android/tree/master/browser-sign-in).
+Example: [Browser sign-in](https://github.com/okta/samples-android/tree/master/browser-sign-in).
```

Comment:
Actually, neither are right as we don't use sign-in as a noun. It would need to be Browser sign-in flow

Reply:
Changed to browser sign-in flow